src/ROUTES.ts

```diff
@@ -624,20 +624,20 @@ const ROUTES = {
     },
     WORKSPACE_WORKFLOWS_APPROVALS_NEW: {
         route: 'settings/workspaces/:policyID/workflows/approvals/new',
-        getRoute: (policyID: string) => `settings/workspaces/${policyID}/workflows/approvals/new` as const,
+        getRoute: (policyID: string, backTo?: string) => getUrlWithBackToParam(`settings/workspaces/${policyID}/workflows/approvals/new` as const, backTo),
```

Comment:
Does this screen need the `backTo` param?

Reply:
It's unused, removing it now 👍